charts/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: web-ui
```

Comment:
that file needs to be in a folder called `web-ui`.

See: https://helm.sh/docs/topics/charts/#the-chart-file-structure